signal/signal.go

```diff
@@ -7,23 +7,23 @@ package signal
 
 const (
 	// StackTrace represents a command to print stack trace.
-	StackTrace = byte(0x1)
+	StackTrace = byte((iota + 1))
```

Comment:
I think it's better https://github.com/google/gops/pull/21/commits/7d9086f8c133c523692146f42b99ba8cb7ef72e6